examples/terraform/digitalocean/output.tf

```diff
@@ -41,10 +41,19 @@ output "kubeone_workers" {
     # corresponding (by name) worker definition
     fra1-1 = {
       replicas        = 3
-      operatingSystem = "ubuntu"
-      droplet_size    = "${var.droplet_size}"
-      region          = "${var.region}"
       sshPublicKeys   = ["${digitalocean_ssh_key.deployer.public_key}"]
+      operatingSystem = "ubuntu"
+
+      operatingSystemSpec = {
```

Comment:
We should try to be as consistent as possible when doing such changes. Please apply a similar change to other Terraform scripts as well.

Reply:
in followup PR, this one was sporadic experiment.

Reply:
Please create a follow-up issue and then I'll lgtm.

Reply:
we already have one: #176

Reply:
This one is about making KubeOne able to parse Terraform booleans and not about making scripts consistent. With this PR merged that issue should be closed. I created #371 instead.